site/en/workflows.md

```diff
@@ -942,13 +1023,39 @@ In this processing step, the text output of the OCR or post-correction can be ev
   </tbody>
 </table>
 
-## Format Conversion (Optional)
+## Generic Data Management (Optional)
 
 OCR-D produces PAGE XML files which contain the recognized text as well as detailed
 information on the structure of the processed pages, the coordinates of the recognized
-elements etc. Optionally, the PAGE XML can be converted to a different output format.
+elements etc. Optionally, the output can be converted, changed or dummy output can be generated. 
+
+### Step 18: Adaptation of Coordinates
+Some processors change the original image by e.g. scaling, deskewing or cropping it and produce alternative images, whose coordinates differ
+from the original images. Nevertheless, all output from the OCR-D-Processors sticks to the coordinates of the original images. If you want to 
```

Comment:
<del>Nevertheless, all output from the OCR-D-Processors sticks to the coordinates of the original images.</del> <ins>However, OCR-D processors adhere to the coordinates of the original image</ins>